docs/concepts/trading-on-vega/positions-margin.md

```diff
@@ -0,0 +1,133 @@
+
+The calculation for a new position, and the amount deducted from collateral to cover margin, is based on all of a trader's open orders. A trader will need enough margin to keep a position open, whether it goes for or against the trader.
+
+The margin set aside for a given position is dictated by the specifics of the position and its exposure. For example, if a trader has a long position on a market, and then reduces the position - their margin requirement doesn't get larger.
+
+:::note Go deeper
+**[Automated cross margining](https://vega.xyz/papers/vega-protocol-whitepaper.pdf#page21)** - Section 6 of the protocol whitepaper.
+:::
+
+## Cross-margining
+Vega's margining system provides automated cross margining. Cross margining, which means gains on one market can be released and used as margin on another, is supported between all markets that use the same settlement asset.
+
+To more closely control how much is risked on a position, it's possible to replicate the effects of isolated margin by using one party (public key) per market.
+
+### Basic margin calculations
+The Vega protocol calculates four margin levels:
+* **Initial level**: The amount that will be transferred from your collateral to be used as margin when an order is placed or trade executed. To avoid a margin search as soon as position is open, the initial margin level is set above the search level:
+  * m<sup>initial</sup> := α<sup>initial</sup> * m<sup>maintenance</sup>, where α<sup>initial</sup> is a constant and α<sup>initial</sup> > α<sup>search</sup>.
+* **Search level**: When the margin balance drops below the search level -- but is still above the maintenance level -- the network will try to allocate an additional amount (up to the current initial margin level) from a trader's collateral, if possible, to be used for margin:
+  * m<sup>search</sup> := α<sup>search</sup> * m<sup>maintenance</sup>, where α<sup>search</sup> is a constant and α<sup>search</sup> > 1.
+* **Release level**: Once a trader's margin balance exceeds the margin release level, the position is considered overcollateralised and funds are released back to collateral so that the margin balance is equal to the current initial margin level:
+  * m<sup>release</sup> := α<sup>release</sup> * m<sup>maintenance</sup>, where α<sup>release</sup> is a constant and α<sup>release</sup> > α<sup>initial</sup>.
+* **Maintenance margin**: This is implied by the risk model and corresponds to the minimum amount required to cover adverse market moves with a given probability level. As soon as the margin balance drops below the maintenance margin level, the position close-out process gets initiated.
+
+:::note Go deeper
+**[Margins and credit risk ↗](https://vega.xyz/papers/margins-and-credit-risk.pdf)** - Section 6 of the protocol whitepaper.
+:::
+
+
+### Calculating the margin on open orders
+The network calculates the largest long / short position. 
```

Comment:
I think it is worth a reminder that "position" on Vega is open volume & open orders, in this case only the open orders that are in the same direction as your opening volume. Or to put another way.

This is not good enough to be a suggestion but:

> The network calculates the what side of your position is riskiest. 
> e.g. You have an open vol of 10 (long) with an open order to buy 5, you also have an order to sell 10. The long is riskiest (10+5).